docs/source/notes/get_start_xpu.rst

```diff
@@ -61,7 +61,8 @@ Now we have all the required packages installed and environment acitvated. Use t
    # (optional) If using torch.compile with inductor/triton, install the matching version of triton
    # Run from the pytorch directory after cloning
    # For Intel GPU support, please explicitly `export USE_XPU=1` before running command.
-   USE_XPU=1 make triton
+   export USE_XPU=1
```

Comment:
I like previous variant more. `USE_XPU=1` is currently needed for `make triton` and old version did emphasize that. While this can be used with `setup.py develop` below, it's not required - XPU environment will be detected automatically if configured properly.